build.py

```diff
@@ -0,0 +1,188 @@
+#! /bin/python3
+import argparse
+import contextlib
+import itertools
+import json
+import os
+import shutil
+import subprocess
+import sys
+import time
+from functools import partial
+from glob import glob
+from pathlib import Path
+from zipfile import ZipFile, ZIP_DEFLATED
+
+printe = partial(print, file=sys.stderr)
+
+sys.path.append('./pylibs')
+try:
+    import watchdog
+except ModuleNotFoundError:
+    pass  # Not mandatory
```

Comment:
```suggestion
    watchdog = None  # Not mandatory
```
and then check if watchdog is None instead of catching a NameError

Reply:
That is also a valid way. I'll do that.